src/wallet/wallet.cpp

```diff
@@ -4204,6 +4204,29 @@ std::shared_ptr<CWallet> CWallet::CreateWalletFromFile(interfaces::Chain& chain,
             return nullptr;
         }
     }
+
+    if (gArgs.IsArgSet("-maxtxfee"))
+    {
```

Comment:
nit: bracket

Reply:
Didn't catch this when moving the code from init.cpp. Sorry!